internal/query_builder.go

```diff
@@ -131,6 +134,19 @@ func (q *queryBuilderImpl) StartTime(minStartTime, maxStartTime time.Time) Query
 	return q
 }
 
+func (q *queryBuilderImpl) CloseTime(minCloseTime, maxCloseTime time.Time) QueryBuilder {
+	CloseTimeQueries := make([]string, 0, 2)
+	if !minCloseTime.IsZero() {
```

Comment:
nit: maybe add some comments behind the reason to append the new string into queries.

Reply:
This is to signify min and max time stamp. Will add the log in the next PR. Thanks